spec/fixtures/balancer_utils.lua

```diff
@@ -99,11 +99,40 @@ local function put_target_endpoint(upstream_id, host, port, endpoint)
   return res, err
 end
 
+local function client_sync_request(proxy_host , proxy_port)
```

Comment:
It might seem an excess, but documenting this function with something along the lines of "`client_sync_request` requires a route with `hosts = { "200.test" }` to sync requests" probably will save some minutes for someone using it in the future. It will save me some time for sure, because I will forget 😄